src/StoryTeller/Model/Persistence/DSL/SentenceMode.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System.Linq;
+using StoryTeller.Model.Persistence.Markdown;
+
+namespace StoryTeller.Model.Persistence.DSL
+{
+    public class SentenceMode : IReaderMode
+    {
+        private readonly Sentence _sentence;
+
+        public SentenceMode(Sentence sentence)
+        {
+            _sentence = sentence;
+        }
+
+        public int Indention { get; set; }
+
+        public IReaderMode Read(int indention, string text)
+        {
+            if (!text.IsTableLine()) return null;
+
+            var values = text.ToTableValues();
```

Comment:
Yep. Just thinking that that would make it a little easier when authoring this stuff by hand